src/homepageExperience/components/HomepageIcons.tsx

```diff
@@ -1,121 +1,111 @@
 import React from 'react'
 
-export const OverviewIcon = props => {
-  return (
-    <svg
-      className="subway-navigation-step-icon"
-      xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 24 24"
-      width="16"
-      height="16"
-    >
-      <path fill="none" d="M0 0h24v24H0z" />
-      <path
-        fill={props.fill}
-        d="M3 18.5V5a3 3 0 0 1 3-3h14a1 1 0 0 1 1 1v18a1 1 0 0 1-1 1H6.5A3.5 3.5 0 0 1 3 18.5zM19 20v-3H6.5a1.5 1.5 0 0 0 0 3H19zM5 15.337A3.486 3.486 0 0 1 6.5 15H19V4H6a1 1 0 0 0-1 1v10.337z"
-      />
-    </svg>
-  )
-}
+export const OverviewIcon = (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    width="16"
+    height="16"
+  >
+    <path fill="none" d="M0 0h24v24H0z" />
+    <path d="M3 18.5V5a3 3 0 0 1 3-3h14a1 1 0 0 1 1 1v18a1 1 0 0 1-1 1H6.5A3.5 3.5 0 0 1 3 18.5zM19 20v-3H6.5a1.5 1.5 0 0 0 0 3H19zM5 15.337A3.486 3.486 0 0 1 6.5 15H19V4H6a1 1 0 0 0-1 1v10.337z" />
+  </svg>
+)
```

Comment:
Just converted it into a simple `svg` variable instead of a function.

Reply:
you'll have to show me how you did this!